pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```
@@ -77,9 +77,21 @@ spec:
                       description: Tags is a map of key/value tags used to select
                         subnets Specifying '*' for a value selects all values for
                         a given tag key.
+                      maxProperties: 30
                       type: object
+                      x-kubernetes-validations:
+                      - message: empty tag keys and values aren't supported
```

```suggestion
                      - message: empty tag keys or values aren't supported
```

nit: this is either/or not both